gradle.properties

```diff
@@ -1,18 +1,4 @@
-# The Gradle daemon aims to improve the startup and execution time of Gradle.
-# When set to true the Gradle daemon is to run the build.
-org.gradle.daemon=true
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-org.gradle.parallel=true
-
-# Enables new incubating mode that makes Gradle selective when configuring projects.
-# Only relevant projects are configured which results in faster builds for large multi-projects.
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:configuration_on_demand
-org.gradle.configureondemand=true
+org.gradle.daemon=false
```

Comment:
In what aspect is the changes in this file an improvement? What is it that will be improved? Build time? Resoursce use? Security?

Reply:
Yes, in this case, this is an improvement for ci-cd scripts. The daemon is convenient only if you reuse the virtual machine to build several projects, but in the current version, you will have a docker to start in which the new daemon will start every time, it turns out that the essence of the daemon is lost and you only spend more time on start the build process.